rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -56,6 +57,11 @@ void Player::play(const PlayOptions & options)
 {
   prepare_publishers();
 
+  if(options.clock_type == "current")
```

Comment:
as @emersonknapp mentioned before I don't think that a string compare is the right thing here. At least an `enum` or if not the actual `rcl_clock_type` makes more sense to me here.